snippets/reject.md

```diff
@@ -3,7 +3,7 @@ title: reject
 tags: array,beginner
 ---
 
-Filters an array's values based on a predicate function, returning only values for which the predicate function returns `true`.
+Filters an array's values based on a predicate function, returning only values for which the predicate function returns `false`.
 
 Use `Array.prototype.filter()` in combination with the predicate function, `pred`, to return only the values for which `pred()` returns `true`.
```

Comment:
I think this needs to be updated, too?

Reply:
update what? I'm sorry, my english is not good, I can't undestand what you mean.

Reply:
I mean this:
```suggestion
Use `Array.prototype.filter()` in combination with the predicate function, `pred`, to return only the values for which `pred()` returns `false`.
```

Reply:
I don't think so, because in code, before pred(), there is a '!'

Reply:
yes, i agree with you.